packages/kotti-button/src/button.vue

```diff
@@ -53,6 +56,7 @@ export default {
 			const classes = [this.type, this.objectClass]
 			if (this.size === 'small') classes.push('sm')
 			if (this.size === 'large') classes.push('lg')
+			classes.push('kt-btn')
```

Comment:
why not add 'kt-btn' to `const classes = [this.type, this.objectClasses, 'kt-btn']` ?

Reply:
Sure we can do it :)

Reply:
but you still didn't